configs/permissions/insights.json

```diff
@@ -1,3 +1,7 @@
 {
-    "*": ["*"]
+    "*": ["*"],
+    "acks": ["read", "write", "*"],
```

Comment:
@dkuc spoke to Rob about this - "acks" isn't customer facing friendly / should the resource type here instead be something like "rules" where "write" permissions give user the ability to enable or disable rules and "read" lets you view rules? Also is it still "rules" or is it now "recommendations"? Recommendations is what is showing up in the cloud.redhat.com UI left nav under Advisor.

Reply:
So we're pretty flexible.  We can change it in way you want.  Here are a few thoughts / concerns first though.

- On the engineering side we use rules/recommendations interchangeably. On the frontend it is recommendations
- Write on recommendations sounds like you can create/update recommendations

I think one point of friction is trying to force semantic meaning on permissions.  Roles are much better for that as they have a title and description.

Reply:
@dkuc maybe let's just use the full word then?